packages/xrpl/snippets/src/multisigning.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  multisign,
+  Client,
+  AccountSet,
+  convertStringToHex,
+  SignerListSet,
+} from '../../dist/npm'
+
+const client = new Client('wss://s.altnet.rippletest.net:51233')
+
+async function multisigning(): Promise<void> {
+  await client.connect()
+  const { wallet: wallet1 } = await client.fundWallet()
+  const { wallet: wallet2 } = await client.fundWallet()
+  const { wallet: walletMaster } = await client.fundWallet()
+  const signerListSet: SignerListSet = {
+    TransactionType: 'SignerListSet',
+    Account: walletMaster.classicAddress,
+    SignerEntries: [
+      {
+        SignerEntry: {
+          Account: wallet1.classicAddress,
+          SignerWeight: 1,
+        },
+      },
+      {
+        SignerEntry: {
+          Account: wallet2.classicAddress,
+          SignerWeight: 1,
+        },
+      },
+    ],
+    SignerQuorum: 2,
+  }
+
+  const signerListResponse = await client.submit(signerListSet, {
+    wallet: walletMaster,
+  })
+  console.log('SignerListSet constructed successfully:')
+  console.log(signerListResponse)
+
+  const accountSet: AccountSet = {
+    TransactionType: 'AccountSet',
+    Account: walletMaster.classicAddress,
+    Domain: convertStringToHex('example.com'),
+  }
+  const accountSetTx = await client.autofill(accountSet, 2)
+  console.log('Transaction set successfully:')
```

Comment:
`Transaction set successfully:` is a bit confusing - I'm not sure what you mean by `set` - I think this rewording might work better
```suggestion
  console.log('AccountSet transaction is ready to be multisigned:')
```